File: python_ToSXs/python/08-mang/58.py
arrInt = [4, 2, 3, 6]

sumEven = 0
sumOdd = 0
sumAll = 0

for elm in arrInt:
    if elm % 2 == 0:
        sumEven += elm
    else:
        sumOdd += elm

sumAll = sumEven + sumOdd

print(f"SumEven: {sumEven}")
print(f"SumOdd: {sumOdd}")
print(f"SumAll: {sumAll}")

File: cpp_e9zMZ/cpp/08-mang/58.cpp
#include <iostream>
using namespace std;

int main() {
    int arrInt[] = {4, 2, 3, 6};
    int sumEven = 0;
    int sumOdd = 0;

    for (int number : arrInt) {
        if (number % 2 == 0) {
            sumEven += number;
        } else {
            sumOdd += number;
        }
    }

    cout << "sumEven = " << sumEven << endl;
    cout << "sumOdd = " << sumOdd << endl;
    cout << "sumAll = " << sumEven + sumOdd << endl;
}